src/main/resources/messages/sv/authenticator/username/authenticate/messages

```diff
@@ -4,4 +4,5 @@ view.authenticate=Logga in
 view.username=Användarnamn
 view.submit=Nästa
 validation.error.username.required=Du måste ange ditt användarnamn
+view.register=Registrera
```

Comment:
No equivalent change in Portuguese resources?

Reply:
This plugin only seems to have en and sv